cli/replicaset/lua/cartridge/get_topology_replicasets_body.lua

```diff
@@ -43,8 +43,18 @@ for _, replicaset in pairs(replicasets) do
 end
 
 local failover_params = require('cartridge').failover_get_params()
+local issues = require('cartridge.issues').list_on_cluster()
+local is_critical = false
+for _, issue in pairs(issues) do
+    if issue.level == 'critical' and issue.instance_uuid == box.info().uuid then
```

Comment:
nitpick: save current instance `uuid` before loop to avoid `box.info` call on every iteration.

Reply:
Thank you! Changed.